packages/instant/src/components/ui/container.tsx

```diff
@@ -27,14 +26,14 @@ export interface ContainerProps {
     backgroundColor?: ColorOption;
     hasBoxShadow?: boolean;
     zIndex?: number;
+    whiteSpace?: string;
     opacity?: number;
 }
 
-const PlainContainer: React.StatelessComponent<ContainerProps> = ({ children, className }) => (
-    <div className={className}>{children}</div>
-);
-
-export const Container = styled(PlainContainer)`
+export const Container =
```

Comment:
Why is this different from how `SlidingAnimation` is defined?

Reply:
because whoops